### ISSUE TITLE
Missing 'Studies included in quantitative synthesis' box?

### ISSUE THREAD

**Issue by befriendabacterium** [nealhaddaway/ROSES_flowchart#2]
Hi Neal,

Just found this - looks like a nice alternative to the PRISMA2020 diagram and its occasionally Rumplestiltskinesque terminology and layout.

I have noticed that the ROSES diagram produced by this code/app is missing the final 'Studies included in quantiative/qualitative/other syntheses' box that is in the example in the paper. Currently it looks like:
![image](https://github.com/nealhaddaway/ROSES_flowchart/assets/42059798/71a2d994-d9e9-437b-a21a-9731f5d9568d)

Whereas I think it maybe is supposed to look like:
![image](https://github.com/nealhaddaway/ROSES_flowchart/assets/42059798/5d757e85-6acf-4479-8362-4182be676cb8)

You'll notice i've changed the text in the last box and shifted the exclusion box to be on the same level as the narrative synthesis box, which i think is more consistent with the rest of the diagram and PRISMA diagrams? Maybe i'm misunderstanding but let me know what you think and whether you could incorporate (please!). It would be great to have to option to specify the number of studies included in the meta-analyses, anyway.

Great work as usual!

Cheers
Matt

**Comment by nealhaddaway** [comment]
Hey Matt - thanks, yeah I hate the PRISMA terms. The ROSES flowchart app is quite behind the PRISMA one in terms of functionality though, Chris has done some amazing stuff on PRISMA.

I think you might be using the systematic map template, which won't have meta-analysis... The second image you linked to doesn't seem to work though..

Thanks!
Neal
________________________________
From: Matt Lloyd Jones ***@***.***>
Sent: 09 February 2024 10:42
To: nealhaddaway/ROSES_flowchart ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nealhaddaway/ROSES_flowchart] Missing 'Studies included in quantitative synthesis' box? (Issue #2)


Hi Neal,

Just found this - looks like a nice alternative to the PRISMA2020 diagram and its occasionally Rumplestiltskinesque terminology and layout.

I have noticed that the ROSES diagram produced by this code/app is missing the final 'Studies included in quantiative/qualitative/other syntheses' box that is in the example in the paper. Currently it looks like:
image.png (view on web)<https://github.com/nealhaddaway/ROSES_flowchart/assets/42059798/71a2d994-d9e9-437b-a21a-9731f5d9568d>

Whereas I think it maybe is supposed to look like:
image.png (view on web)<https://github.com/nealhaddaway/ROSES_flowchart/assets/42059798/5d757e85-6acf-4479-8362-4182be676cb8>

You'll notice i've changed the text in the last box and shifted the exclusion box to be on the same level as the narrative synthesis box, which i think is more consistent with the rest of the diagram and PRISMA diagrams? Maybe i'm misunderstanding but let me know what you think and whether you could incorporate (please!). It would be great to have to option to specify the number of studies included in the meta-analyses, anyway.

Great work as usual!

Cheers
Matt

**Comment by befriendabacterium** [comment]
Hi Neal,

Thanks for quick response! Ah sorry my brain is fried from a long week making PRISMA2020 diagrams with Chris and your R package (which makes things much easier but I've still been dreaming about flow diagrams...).

Ah sorry! Actually, I just hadn't seen the 'Select type of synthesis' box! When I select 'quantitative' I pretty much get what I want. Though shouldn't the 'Studies not included in further synthesis' be parralel to the narrative synthesis one (i.e. studies excluded from quant synth?). I've added the image I meant to add before to demonstrate in an edit to my original question above.

Sort of glad to here you say the ROSES is a quite behind PRISMA202 having spent all week making the latter lol. Though I still think your terminology/reporting is much clearer in a lot of ways (e.g. reports vs studies), and as far as I can tell you cannot yet incorporate the extra inclusion/exclusion steps (after critical appraisal, narrative synthesis, quantitative synthesis) in the PRISMA2020 R package? 

Cheers
Matt

**Comment by nealhaddaway** [comment]
Heya,

No worries! Hmm - no, I think we wanted the narrative synthesis box to be slightly lower deliberately, although my memory is sketchy here, and I can't remember exactly why...

And yeah, exclusion reasons and databases are less developed in ROSES. I should try to port some of Chris' changes over but I'm swamped. Not sure many people are using it, too - I don't have analytics for the app.. maybe I should...

Cheers!
Neal
________________________________

**Comment by befriendabacterium** [comment]
Ah ok fair!

Yeah I had a look at implementing a few changes/additions I wanted myself but DiagrammeR doesn't allow you to easily edit an object once it's created from my experience (unlike ggplot's + feature for example). I also briefly tried editing Chris' source code but got some weird errors with hidden functions that appear to be in the PRISMA2020 package but not immediately accessible to users (e.g. 'PRISMA_get_height_'). His implementation also seems quite different to yours although you both use DiagrammeR, so it might be easier for Chris to incorporate afresh than port over?!

**Comment by nealhaddaway** [comment]
Yeah DiagrammeR is a major pain. I think Chris mostly rewrote my code to make his changes, yeah. I don't think incremental edits are really worth the effort with DiagrammeR... If Chris has time that would be great! Big ask, though 😉
________________________________

**Comment by befriendabacterium** [comment]
Lol i've already asked for 4 feature requests for PRISMA2020 this week from poor @chriscpritchard (sorry mate!). Either I owe him a lot of coffees and/or it's time for another hackathon soon...

**Comment by nealhaddaway** [comment]
haha

Yes - plans are afoot...
________________________________
